PLAYER/GetIsPlayerDrivingOnHighway.md

```diff
@@ -0,0 +1,32 @@
+---
+ns: PLAYER
+aliases: ["0x5FC472C501CCADB3"]
+---
+## GET_IS_PLAYER_DRIVING_ON_HIGHWAY
+
+```c
+// 0x5FC472C501CCADB3 0x46E7E31D
+BOOL GET_IS_PLAYER_DRIVING_ON_HIGHWAY(Player playerId);
+```
+
+```
+Appears once in "re_dealgonewrong"  
+```
+
+## Parameters
+* **playerId**: The local index of the player to check (see [PLAYER_ID](#_0x4F8644AF03D0E0D6))
+
+## Return value
+Returns a boolean value representing if the player is driving on a highway.
+
+## Examples
+```lua
+-- Get and store the local player index
+local playerId = PlayerId()
+
+if GetIsPlayerDrivingOnHighway(playerId) then
```

Comment:
You could add JS/C# support too !

Reply:
I don't control support for the natives, simply just contributed to the documentation.